imap_processing/cdf/utils.py

```diff
@@ -36,7 +40,9 @@ def calc_start_time(shcoarse_time: int):
 
 
 def write_cdf(
-    data: xr.Dataset, description: str = "", mode: str = "", directory: str = ""
+    data: xr.Dataset,
+    description: str = "",
+    directory: Optional[Path] = None,
```

Comment:
This too will go away?

Reply:
I am leaving it for now in case someone wants to explicitly put their file somewhere other than the base data directory. We can remove it later if we want, but I did add an explicit test to make sure this worked.